style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-left: 24px;
+  padding-right: 24px;
+  margin-top: 10px;
+  margin-bottom: 100px;
+  flex-direction: row;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline {
+  background-image: url("./image/headlinebackground.png");
```

Comment:
- The background image is not well aligned as compared to that on the Figma. kindly check the Figma design to improve on this to make your design more appealing as seen below:

![image](https://github.com/abenisamuel11/mobile-set-up-season2/assets/59397159/32700ca7-0456-4f38-ab8e-26e64c65fa80)


![image](https://github.com/abenisamuel11/mobile-set-up-season2/assets/59397159/8bb826e2-5328-405b-8e65-a7fcbb22b6bf)